src/audio/module_adapter/module/modules.c

```diff
@@ -295,10 +303,16 @@ static int modules_free(struct processing_module *mod)
 	rfree(md->mpd.in_buff);
 	rfree(md->mpd.out_buff);
 
+#if 0
+	/*
+	 * Module unloading isn't implemented yet, it has to be done from a
+	 * dedicated IPC, not when stopping streams.
+	 */
```

Comment:
Module unloading come from ref counting users based on running pipelines.

Reply:
yes, forgot to update the comment, fixing, thanks